docs/generated/cli/show.md

```diff
@@ -23,10 +23,10 @@ Show all projects in the workspace:
  nx show projects
 ```
 
-Show all projects with names starting with "api-". The pattern option is useful to see which projects would be selected by run-many.:
+Show all projects with names starting with "api-". The "projects" option is useful to see which projects would be selected by run-many.:
 
 ```shell
- nx show projects --pattern=api-*
+ nx show projects -p api-*
```

Comment:
Can we use `--projects` as the example?

Using the alias isn't good for someone new

Reply:
We can.... I think it looks silly when you spell out --projects for this one though